mirror/images.txt

```diff
@@ -9,12 +9,18 @@
 # ceph-csi devel
 docker.io/rook/ceph:v1.8.2	rook/ceph:v1.8.2
 
+
```

Comment:
please remove the additional empty lines. Just a single empty line between section suffices.

Reply:
you marked this as resolved, but I still see the additional empty lines in this commit?